testing/tests/ui/elif.stderr

```diff
@@ -1,5 +1,5 @@
 error: unknown `elif` keyword; did you mean `else if`?
-       problems parsing template source at row 1, column 16 near:
+       failed to parse template source at row 1, column 16 near:
```

Comment:
Note that this doesn't have the shorter location information as suggested in my other comment.

Reply:
If there is no file name (for error coming from the parser, which isn't handled in here), I preferred to leave it as is.